mappings/net/minecraft/client/render/BufferBuilder.mapping

```diff
@@ -9,6 +21,11 @@ CLASS net/minecraft/class_287 net/minecraft/client/render/BufferBuilder
 	FIELD field_1558 currentElement Lnet/minecraft/class_296;
 	FIELD field_1565 format Lnet/minecraft/class_293;
 	FIELD field_1567 drawMode I
+	FIELD field_20774 parameters Ljava/util/List;
+	FIELD field_20775 lastParameterIndex I
+	FIELD field_20776 buildStart I
+	FIELD field_20777 drawStart I
```

Comment:
`drawStart` acts as a `drawLimit` most of the time.  Alternatively, it could be `nextDrawStart` Its only used as the start of the current draw during `pollArrayDrawParameter` and then is immediately updated to point to the start of the next draw.

Reply:
`nextDrawStart` should be better.